Google.py

```diff
@@ -18,13 +18,13 @@ class Google:
     def __init__(self, googlers):
         """ Creates Google company.
 
-        Calls __create_company method which implements manager hierarchy tree based on the list of
+        Calls __create_company method that implements manager hierarchy tree based on the list of
         Googlers and their attributes.
 
         Args:
             googlers: A list of Googler objects.
         """
-        self.ceo = self.__create_company(googlers)
+        self.company = self.__create_company(googlers)
```

Comment:
Why the two underscores? is that a style rule thing>

Reply:
https://www.geeksforgeeks.org/private-methods-in-python/ Yeah Python has no such thing as private methods like Java but double underscore indicates it should only be used within the class.. in this case, to create the dictionary company attribute.

Reply:
cool, thanks for sharing!